platform/android/detect.py

```diff
@@ -158,7 +159,7 @@ def configure(env):
     env["CXX"] = compiler_path + "/clang++"
     env["AR"] = compiler_path + "/llvm-ar"
     env["RANLIB"] = compiler_path + "/llvm-ranlib"
-    env["AS"] = bin_utils_path + "/as"
+    env["AS"] = compiler_path + "/clang"
```

Comment:
`bin_utils` and `bin_utils_path` could be removed, they were only defined for this.